grails-app/domain/com/secured/auth/commands/UserCommand.groovy

```diff
@@ -0,0 +1,44 @@
+package com.secured.auth.commands
```

Comment:
Please move command objects to package
/src/main/groovy/com/secured/user 
and implemnt from Validateable to have validatable command 
Like:  class XXXCO implements Validateable {